### DISCUSSION TITLE
Does Sign in button meet accessibility standards for contrast ratio in top blue banner?

### DISCUSSION THREAD

**Discussion by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#526] [Announcements]
Sign in button in the top blue banner of va.gov may not be accessible in the default state, and almost certainly isn't accessible in the hover state in terms of contrast.

![image](https://user-images.githubusercontent.com/88347129/130148823-23ef60e1-65ed-465e-b3e6-44b101f104be.png)

![image](https://user-images.githubusercontent.com/88347129/130148769-a466609f-8c25-4f27-a288-55fa94d762c4.png)

**Comment by 1Copenut** [comment]
Our medium blue Sign In button has a background color #0071BB and the dark blue header is #112e51. This gives us a contrast ratio of 2.66:1. We will need to make this a minimum 3:1 contrast ratio to meet [SC 1.4.11 Non-text Contrast Ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast).

---
https://whocanuse.com/?b=112e51&c=0071bb&f=20&s=

![Screen Shot 2021-08-20 at 10 49 51 AM](https://user-images.githubusercontent.com/934879/130260057-1100974f-042c-46ab-a2a3-b551a782d016.png)

**Comment by rtwell** [comment]
We'll tackle this in tandem with the optimized mobile header work.
